src/test/java/com/exasol/adapter/dialects/rls/AbstractRowLevelSecurityIT.java

```diff
@@ -68,55 +68,55 @@ private static void uploadAdapterScript() {
     }
 
     private static void registerAdapterScript() {
-        final ExasolSchema schema = factory.createSchema("SCHEMA_FOR_RLS_ADAPTER_SCRIPT");
+        final ExasolSchema schema = objectFactory.createSchema("SCHEMA_FOR_RLS_ADAPTER_SCRIPT");
         final String content = "%scriptclass com.exasol.adapter.RequestDispatcher;\n" //
                 + "%jar /buckets/bfsdefault/default/" + ROW_LEVEL_SECURITY_JAR_NAME_AND_VERSION + ";\n";
         adapterScript = schema.createAdapterScript("RLS_ADAPTER", AdapterScript.Language.JAVA, content);
     }
 
     private static void createConnectionDefinition() {
-        connectionDefinition = factory.createConnectionDefinition("RLS_CONNECTION", "jdbc:exa:localhost:8888",
-                container.getUsername(), container.getPassword());
+        connectionDefinition = objectFactory.createConnectionDefinition("RLS_CONNECTION", "jdbc:exa:localhost:8888",
```

Comment:
shouldn't we use here `EXASOL.getDefaultInternalDatabasePort()` instead of the `8888` port?

Reply:
Right. I am going to change that.